tests/foreman/ui/test_hw_model.py

```diff
@@ -56,14 +54,15 @@ def test_create_negative_with_blank_name(self, name):
 
         """
         with Session(self.browser) as session:
-            make_hw_model(session, name=name)
-            error = session.nav.wait_until_element(
-                common_locators['name_haserror'])
-            self.assertIsNotNone(error)
+            for name in '', '   ':
```

Comment:
Hm... How about combining this test with the previous one by using `invalid_values_list()`? I saw you've already followed this approach in another modules